modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/CreationMenu.js

```diff
@@ -22,13 +22,70 @@ import React, {useContext, useState} from 'react';
 import FrontendDataSetContext from '../../FrontendDataSetContext';
 import {triggerAction} from '../../utils/actionItems/index';
 
-function CreationMenu({inEmptyState, primaryItems}) {
+const EMPTY_STATE_BUTTON_PROPS = {
+	'aria-label': null,
+	'className': null,
+	'displayType': 'secondary',
+	'title': null,
+};
+
+const DropDown = ({inEmptyState, primaryItems}) => {
 	const frontendDataSetContext = useContext(FrontendDataSetContext);
 
 	const {loadData} = frontendDataSetContext;
 
 	const [active, setActive] = useState(false);
 
+	return (
+		<ClayDropDown
+			active={active}
+			onActiveChange={setActive}
+			trigger={
+				<ClayButton
+					aria-label={Liferay.Language.get('new')}
+					className="nav-btn nav-btn-monospaced"
+					title={Liferay.Language.get('new')}
+					{...(inEmptyState ? EMPTY_STATE_BUTTON_PROPS : {})}
+				>
+					{inEmptyState ? (
+						Liferay.Language.get('new')
+					) : (
+						<ClayIcon symbol="plus" />
```

Comment:
If we are able to update the FDS MT, this will turn into a "new" button like in the empty state